CMakeLists.txt

```diff
@@ -45,10 +45,12 @@ find_package(OpenGL REQUIRED)
 find_package(GLEW REQUIRED)
 find_package(glfw3 REQUIRED)
 find_package(glm REQUIRED)
+find_package(OpenMP)
 
 target_link_libraries(FunGame PRIVATE glfw)
 target_link_libraries(FunGame PRIVATE OpenGL::GL)
 target_link_libraries(FunGame PRIVATE GLEW::GLEW)
+#target_link_libraries(FunGame PUBLIC OpenMP::OpenMP_CXX)
```

Comment:
this is wrong

Reply:
clang seemes to be complaining. Do you know why?

Reply:
no